airbyte-server/src/main/java/io/airbyte/server/ServerApp.java

```diff
@@ -80,7 +80,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ServerApp.class);
   private static final int PORT = 8001;
-
+  private static final AirbyteVersion KUBE_SUPPORT_FOR_AUTOMATIC_MIGRATION = new AirbyteVersion("0.26.5-alpha");
```

Comment:
let's leave a comment here explaining why this is the case for Kube i.e. before this version, the airbyte db would erase state upon termination, so we cannot do automatic migrations.